howso/hierarchy.amlg

```diff
@@ -356,14 +232,55 @@
 			;name to load (without extension)
 			filename ""
 			;{type ["string" "list"] values "string"}
-			;name path of trainee to load
-			trainee ""
+			;list of strings specifying the user-friendly path of the child subtrainee to load
+			;including the name of the child as the last value in the path
+			path (null)
+			;{type "string" required (true)}
+			;unique id of child trainee to load
+			child_id (null)
```

Comment:
if child_id is required, is there any need for path?

Reply:
Same goes for other non-create methods

Reply:
both are required for load - path to specify where to load, and id so that the trainee_id is specified and can be used as the parent in case user decides to load subtrainees of this one